api/app/src/routes/medical/schemas/fhir.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+
+const typeSchema = z.enum([
+  "document",
+  "message",
+  "transaction",
+  "transaction-response",
+  "batch",
+  "batch-response",
+  "history",
+  "searchset",
+  "collection",
+]);
+
+export const bundleEntrySchema = z.object({
+  resourceType: z.enum(["Bundle"]),
+  id: z.string().optional(),
```

Comment:
I've been trying to use `.nullish()` instead of `.optional()`, just because its more flexible with api clients sending `null` or omitting the prop altogether when they don't want to set it.